introduction/modules/modules.md

```diff
@@ -1,42 +1,227 @@
 # Available modules
 
-## Documents
+## Core
+
+### Audit
+
+Audit provides a framework to record and store actions performed by users. Certain actions in the other modules of
+Valtimo, like completing a task or uploading a file will be audited by default. It is possible to audit other actions
+by using this feature in the implementation code. The audit trail is available by case to show the history of
+actions for that case.
+
+### Connector
+
+Connectors are Valtimo configurable components that can be used to connect to external systems. This module provides a
+framework to create, configure, and use connectors. Certain Valtimo modules include their own connectors that will be
+automatically available for configuration when starting a Valtimo instance.
+
+### Contract
+
+Contract contains certain interfaces and events that Valtimo uses in several other modules. Apart from this, contract
+sanitizes endpoint output when exceptions occur. A liquibase runner is available that allows other modules to define
+liquibase configuration that is run on startup. Finally, contract include several helper classes, like the
+`QueryDialectHelper` bean that can be used in criteria queries to add JSON column clauses to the query.
+
+### Core
+
+Core contains and builds upon the Camunda process engine that enables Valtimo to run processes. Several endpoint and
+functions are available that provide Task and Process data. It also includes choice fields. A generic way to create
```

Comment:
I see Task and Process are capitalized. I don't agree or disagree with this, but I think we should think of some guidelines on what (and what not) to capitalize. Right now, I would not know exactly why and when

Reply:
I'll turn them into lowercase for now, so at least it's consistent